server/src/controllers/Events/resolver.ts

```diff
@@ -58,6 +54,11 @@ import {
 } from '../../util/calendar';
 import { updateWaitlistForUserRemoval } from '../../util/waitlist';
 import { redactSecrets } from '../../util/redact-secrets';
+import {
+  getChapterUnsubscribeToken,
+  getUnsubscribeOptions,
+  NotificationContextText,
```

Comment:
```suggestion
  getChapterUnsubscribeOptions,
  getEventUnsubscribeOptions,
```
We shouldn't be exporting helper functions from event email, just the final two functions that give us the text we want.